src/qibo/backends/__init__.py

```diff
@@ -1,35 +1,63 @@
 import os
+from pkgutil import iter_modules
 from qibo import config
 from qibo.config import raise_error, log, warnings
 from qibo.backends.numpy import NumpyDefaultEinsumBackend, NumpyMatmulEinsumBackend
 from qibo.backends.tensorflow import TensorflowCustomBackend, TensorflowDefaultEinsumBackend, TensorflowMatmulEinsumBackend
 
 
+def _check_availability(module_name):
```

Comment:
This can also be a `@staticmethod` of the `Backend` class since we do not use it anywhere outside this class.